### PR TITLE
Blog on interrupts, softirqs and softnets in Linux

### PR DESCRIPTION
**Pull request by sashwathn**
Please review and Approve

### CODE REVIEW

**Review by andrewm4894 — APPROVED** [review]

**Review by andrewm4894 — CHANGES_REQUESTED** [review]
Should there be two posts here?

File: blog/2023-05-02-understanding-interrupts-softirqs-and-softnet-in-linux/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Monitoring interrupts, softirqs, and softnet can help you identify performance issues and troubleshoot problems in cloud VMs. By monitoring these metrics, you can identify potential issues related to hardware devices, software performance, and network traffic. This can help you take proactive steps to optimize system performance and avoid issues that could impact your VMs.
+
+## How to Monitor Interrupts, Softirqs, and Softnet with Netdata?
+
+Netdata is a powerful real-time performance monitoring tool that can provide insights into system performance, including interrupts, softirqs, and softnet. Netdata can collect and display metrics on a wide variety of system resources, including CPU usage, memory usage, network traffic, and disk I/O, and many more.
+
+To monitor interrupts, softirqs, and softnet in Netdata for cloud VMs, you can navigate to the `System Overview` dashboard and look for the relevant charts. These charts can show you the current values for these metrics, as well as historical trends over time.
+
+By monitoring interrupts, softirqs, and softnet per core at the CPUs section, you can identify which CPU cores are experiencing the highest levels of activity.
+
```
Comment (cakrit) [inline blog/2023-05-02-understanding-interrupts-softirqs-and-softnet-in-linux/index.md:104]:
```suggestion
![image](https://user-images.githubusercontent.com/43294513/235747305-9f5c8d3a-4e76-43b1-882c-3b1307f1dd41.png)
```

**Review by cakrit — CHANGES_REQUESTED** [review]

**Comment by sashwathn** [comment]
@andrewm4894 / @cakrit : Not sure how the context-switching blog came up here. I created a new branch after pushing the other one - weird.

**Review by cakrit — APPROVED** [review]

**Comment by sashwathn** [comment]
Closing the PR and recreating another one..

**Comment by sashwathn** [comment]
@andrewm4894 : I fixed the files being committed.

**Pull request merged**